Users can see own initiatives

hyosunko opened the pull request.

Commit:
button added on users page

Commit:
nav bar implemented logo

Commit:
Implemented and styled navbar, added attributes to new user form

Commit:
deleted unnecessary files, cleaned up white spaces, rspec for users pass

Commit:
Finished integrated testing, deleted show.html.erb and resolved jason's comments

Commit:
resolved whitespace issues and resolved div tag issues, removed inline styling from users/index.html.erb

Commit:
impletmented files.insertFinalNewline to files

kevinyu05 committed:
added css for checkbox and dropdown, added dropdown functionality, added required password field, added logic to user model, testing WIP

kevinyu05 committed:
login page functional, WIP testing, applied css for login page

kevinyu05 committed:
feature test passed, rubocop passed, added support file to aid in feature tests

hyosunko committed:
add log out functionality, and clean up login page, rubocop & rspec test passed

hyosunko committed:
change profile for heroku change

kevinyu05 committed:
user menu only appears for Admin, user management functionality available for admin, flash message for non-admin implemented, testing WIP

kevinyu05 committed:
feature test passed, rubocop passed

hyosunko committed:
add admin test, modify seed data

hyosunko committed:
fixed missing bracket %>

hyosunko committed:
fixed indentation & add test for sign out

hyosunko committed:
fix indentations

hyosunko committed:
WIP add user scope on initiatives

hyosunko committed:
add warning message if user enter other's initiative routes, and add css for each controller routes in nav bar

casto101 commented on app/controllers/initiatives_controller.rb:
```diff
@@ -40,19 +40,24 @@ def destroy
+    @initiative = current_user.initiatives.find_by_id(params[:id])
+    
+    if @initiative.nil?
+      flash[:notice] = "You entered wrong routes"
```
This notice will be shown to the user, but the user probably doesn't know what routes are--that's internal to Rails. Is there a more user-friendly message we could show here to improve the user experience?

hyosunko replied:
modified flash message for invalid route

casto101 reviewed (approved):
LGTM with one comment.

hyosunko committed:
changed flash message for invalid route.
deleted unnecessary comments from appplication controller.
Coauthered_by: Kevin Yu<kevinyu05@gmail.com>

hyosunko committed:
fixed navbar font color to follow controller_name
Coauthered_by: Kevin Yu<kevinyu05@gmail.com>
